src/webui/btjson.cpp

```diff
@@ -711,12 +727,37 @@ QVariantMap getTranserInfoMap()
 {
     QVariantMap map;
     BitTorrent::SessionStatus sessionStatus = BitTorrent::Session::instance()->status();
+    BitTorrent::CacheStatus cacheStatus = BitTorrent::Session::instance()->cacheStatus();
     map[KEY_TRANSFER_DLSPEED] = sessionStatus.payloadDownloadRate();
     map[KEY_TRANSFER_DLDATA] = sessionStatus.totalPayloadDownload();
     map[KEY_TRANSFER_UPSPEED] = sessionStatus.payloadUploadRate();
     map[KEY_TRANSFER_UPDATA] = sessionStatus.totalPayloadUpload();
     map[KEY_TRANSFER_DLRATELIMIT] = BitTorrent::Session::instance()->downloadSpeedLimit();
     map[KEY_TRANSFER_UPRATELIMIT] = BitTorrent::Session::instance()->uploadSpeedLimit();
+
+    quint64 atd = BitTorrent::Session::instance()->getAlltimeDL();
+    quint64 atu = BitTorrent::Session::instance()->getAlltimeUL();
```

Comment:
Look at the top. You already holde the session instance in a variable. (`sessionStatus`)